Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1410,3 +1256,223 @@ const CGameListCtrl *CFrame::GetGameListCtrl() const
+		}
+		else
+		{
+			g_Config.bCopyEFBToTexture = !g_Config.bCopyEFBToTexture;
+		}
+	}
+	else if (IsHotkey(event, HK_TOGGLE_FOG))
+	{
+		OSDChoice = 4;
+		g_Config.bDisableFog = !g_Config.bDisableFog;
+	}
+	else if (IsHotkey(event, HK_TOGGLE_THROTTLE, true))
+	{
+		Core::SetIsFramelimiterTempDisabled(true);
+	}
+	else if (IsHotkey(event, HK_DECREASE_FRAME_LIMIT))
+	{
+		if (--SConfig::GetInstance().m_Framelimit > 0x19)
+			SConfig::GetInstance().m_Framelimit = 0x19;
+	}
+	else if (IsHotkey(event, HK_INCREASE_FRAME_LIMIT))
+	{
+		if (++SConfig::GetInstance().m_Framelimit > 0x19)
+			SConfig::GetInstance().m_Framelimit = 0;
+	}
+	else if (IsHotkey(event, HK_SAVE_STATE_SLOT_SELECTED))
+	{
+		State::Save(g_saveSlot);
+	}
+	else if (IsHotkey(event, HK_LOAD_STATE_SLOT_SELECTED))
+	{
+		State::Load(g_saveSlot);
+	}
+	else if (IsHotkey(event, HK_DECREASE_DEPTH))
+	{
+		if (--g_Config.iStereoDepth < 0)
+			g_Config.iStereoDepth = 0;
+	}
+	else if (IsHotkey(event, HK_INCREASE_DEPTH))
+	{
+		if (++g_Config.iStereoDepth > 100)
+			g_Config.iStereoDepth = 100;
+	}
+	else if (IsHotkey(event, HK_DECREASE_CONVERGENCE))
+	{
+		if (--g_Config.iStereoConvergence < 0)
+			g_Config.iStereoConvergence = 0;
+	}
+	else if (IsHotkey(event, HK_INCREASE_CONVERGENCE))
+	{
+		if (++g_Config.iStereoConvergence > 500)
+			g_Config.iStereoConvergence = 500;
+	}
+
+	else
+	{
+		for (int i = HK_SELECT_STATE_SLOT_1; i < HK_SELECT_STATE_SLOT_10; ++i)
+		{
+			if (IsHotkey(event, i))
+			{
+				wxCommandEvent slot_event;
+				slot_event.SetId(i + IDM_SELECT_SLOT_1 - HK_SELECT_STATE_SLOT_1);
+				CFrame::OnSelectSlot(slot_event);
+			}
+		}
+
+		unsigned int i = NUM_HOTKEYS;
+		if (!SConfig::GetInstance().m_LocalCoreStartupParameter.bRenderToMain || TASInputHasFocus())
+		{
+			for (i = 0; i < NUM_HOTKEYS; i++)
+			{
+				if (IsHotkey(event, i))
+				{
+					int cmd = GetCmdForHotkey(i);
+					if (cmd >= 0)
+					{
+						wxCommandEvent evt(wxEVT_MENU, cmd);
+						wxMenuItem *item = GetMenuBar()->FindItem(cmd);
+						if (item && item->IsCheckable())
+						{
+							item->wxMenuItemBase::Toggle();
+							evt.SetInt(item->IsChecked());
+						}
+						GetEventHandler()->AddPendingEvent(evt);
+						break;
+					}
+				}
+			}
+		}
+		// On OS X, we claim all keyboard events while
+		// emulation is running to avoid wxWidgets sounding
+		// the system beep for unhandled key events when
+		// receiving pad/Wiimote keypresses which take an
+		// entirely different path through the HID subsystem.
+#ifndef __APPLE__
+		// On other platforms, we leave the key event alone
+		// so it can be passed on to the windowing system.
+		if (i == NUM_HOTKEYS)
+			event.Skip();
+#endif
+	}
+
+	// Actually perform the Wiimote connection or disconnection
+	if (Core::GetState() != Core::CORE_UNINITIALIZED)
+	{
+		if (WiimoteId >= 0)
+		{
+			wxCommandEvent evt;
+			evt.SetId(IDM_CONNECT_WIIMOTE1 + WiimoteId);
+			OnConnectWiimote(evt);
+		}
+
+		if (g_Config.bFreeLook)
```

Comment:
It would be nice to remove it eventually, but I don't think that the current hotkey setup can do the "right click for pan" feature that the current "Free Look" checkmark box allows.